application/F3DStarter.cxx

```diff
@@ -79,6 +81,23 @@ class F3DStarter::F3DInternals
       .setCurrentAsDefault();
   }
 
+  static bool HasHDRIExtension(const std::string file)
+  {
+    std::string ext = fs::path(file).extension();
+    std::transform(ext.begin(), ext.end(), ext.begin(), ::toupper);
+#if F3D_MODULE_EXR
+    if (ext == ".EXR")
```

Comment:
Any image can be used (.png, .jpg, .tif, ...)  
I got you don't have access to the image reader factory, but it's probably not that difficult to just add a `f3d::image::supports(const std::string&)` right away.

Reply:
I will not add a new feature once the master/release split has occured.

Reply:
And only .hdr and .exr matters for this interaction (drop). Will probably be improved in the future as stated on discord.